python/lsst/cp/verify/verifyBias.py

```diff
@@ -45,6 +54,32 @@ class CpVerifyBiasTask(CpVerifyStatsTask):
     ConfigClass = CpVerifyBiasConfig
     _DefaultName = 'cpVerifyBias'
 
+    def imageStatistics(self, exposure, uncorrectedExposure, statControl):
+        # Docstring inherited
+        outputStatistics = super().imageStatistics(exposure, uncorrectedExposure, statControl)
+
+        # These should be a config item, probably.
```

Comment:
You mean the statistic as well as the box size?  Either fix this or remove the comment.

Reply:
Just the box size.  I've removed the comment as it needed to be made a config to pass the unit tests (the default box size is too large for the mocked image).